src/FormCanvas-Core/Bezier2Segment.class.st

```diff
@@ -389,7 +387,6 @@ Bezier2Segment >> printOn: aStream [
 Bezier2Segment >> roundTo: quantum [
 	super roundTo: quantum.
 	via := via roundTo: quantum.
```

Comment:
Dot at end of last statement :-)

Reply:
There are lots of extra dots. Let's not fix them in the same PR